packages/react-core/src/demos/Button.md

```diff
@@ -0,0 +1,75 @@
+---
+id: Button
+section: components
+---
+
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+
+## Demos
+
+### Progress button
+
+```ts
+import React from 'react';
+import { Form, FormGroup, ActionGroup, InputGroup, TextInput, Button } from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+
+const ProgressButton = () => {
+  const [loginState, setLoginState] = React.useState<'notLoggedIn' | 'loading' | 'loggedIn'>('notLoggedIn');
+  const actionButton = {
+    notLoggedIn: (
+      <Button
+        variant="primary"
+        onClick={() => {
+          setLoginState('loading');
+          setTimeout(() => {
+            setLoginState('loggedIn');
+          }, 3000);
+        }}
+      >
+        Link account and log in
+      </Button>
+    ),
+    loading: (
+      <Button variant="primary" aria-label="Loading..." isLoading>
+        Linking account
+      </Button>
+    ),
+    loggedIn: (
+      <Button variant="primary" icon={<CheckCircleIcon />}>
+        Logged in
+      </Button>
+    )
+  };
+
+  return (
+    <Form>
+      <FormGroup label="Username" isRequired fieldId="progress-button-initial-login">
+        <InputGroup>
```

Comment:
No need for an `<InputGroup>` unless you want to have a group of form elements/buttons/dropdowns you want to appear beside one another.

Reply:
Thanks for the explanation! removed it